widgets/DateTransform-transform.json

```diff
@@ -0,0 +1,64 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "configuration-groups": [
+    {
+      "label": "Date Transform Configuration",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Source Field Name",
+          "name": "sourceField",
+          "widget-attributes": {
+            "width": "large"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Source Field Date Format",
```

Comment:
If we are accepting only a fixed number of formats than this should be a drop down, That will avoid errors